test/e2e/workspacetype/controller_test.go

```diff
@@ -58,28 +58,40 @@ func TestClusterWorkspaceTypes(t *testing.T) {
 					return server.orgKcpClient.TenancyV1alpha1().ClusterWorkspaces().Get(ctx, workspace.Name, metav1.GetOptions{})
 				})
 
-				err = server.orgExpect(workspace, ready)
-				require.NoError(t, err)
+				require.Eventually(t, func() bool {
+					workspace, err = server.orgKcpClient.TenancyV1alpha1().ClusterWorkspaces().Get(ctx, workspace.Name, metav1.GetOptions{})
```

Comment:
This get can never fail. And if it does, fail hard: `require.NoError(err)`.

Reply:
Just moved from expect -> eventually since they are the same